tests/topotests/lib/common_config.py

```diff
@@ -677,7 +677,17 @@ def generate_support_bundle():
     for rname, rnode in router_list.items():
         logger.info("Generating support bundle for {}".format(rname))
         rnode.run("mkdir -p /var/log/frr")
-        bundle_log = rnode.run("python2 /usr/lib/frr/generate_support_bundle.py")
+
+        # Support both python2 and 3 for the time being
+        if sys.version_info[0] > 2:
+            bundle_log = rnode.run(
+                "/usr/bin/python3 /usr/lib/frr/generate_support_bundle.py"
+            )
+        else:
+            bundle_log = rnode.run(
+                "/usr/bin/python2 /usr/lib/frr/generate_support_bundle.py"
+            )
+
```

Comment:
Had a discussion with David and here is what I would prefer:
1) Ignore python2. It's EoL and I would not want to verify the compatibility anymore with python2 in the near future.
2) Skip the check for python 2 or 3 and **always** use python3 (and refer it to python3, not generic python which could be either.
3) Use the `env` and don't hard-code the python3 paths as qlyoung mentioned. (As a alternative, we should consider adding the python3 shebang in the generate_support_bundle.py script and make it executable).